docs/Job-reference.md

```diff
@@ -4286,3 +4286,46 @@ job {
 ```
 
 (since 1.26)
+
+# Workflow Definitions
+
+## Groovy CPS DSL
+
+```
+job(type: Workflow) {
+    definition {
+        cps {
+            script(String script)
+            sandbox(boolean sandbox = true)
+        }
+    }
+}
+```
+
+Defines a Groovy CPS DSL definition.
+
+```
+def flow = '''node {
+  git url: 'https://github.com/jglick/simple-maven-project-with-tests.git'
+  def mvnHome = tool 'M3'
+  sh "${mvnHome}/bin/mvn -B verify"
+}'''
```

Comment:
Reading the script from a separate file is supported, see the next example below (https://github.com/jenkinsci/job-dsl-plugin/pull/361/files#diff-28eee2ed320096a6f8af8a46a5140726R4324).